demos/quick-start/docker/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apk add --no-cache apache2-utils \
                        tini \
                        openssh \
                        openssl \
+    && apk upgrade --no-cache libxslt \
```

Comment:
Can we do [this](https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Upgrade_a_Running_System) instead. We should also probably do this before any custom package install (between line 21 and 23) to take advantage of caching.